Plugin/CartIntercept.php

```diff
@@ -42,35 +42,42 @@ public function __construct(
         $this->swatchModel = $swatchModel;
     }
 
+    private function isLoEnabled($storeId)
+    {
+        return $this->sailthruSettings->isAbandonedCartEnabled($storeId) == $this->sailthruSettings::LO_ABANDONED_CART_ENABLED;
+    }
+
     public function _gate(Cart $cart)
     {
         $storeId = $cart->getQuote()->getStoreId();
         if ($this->sailthruSettings->isAbandonedCartEnabled($storeId)) {
-            $this->client = $this->client->getClient(true, $storeId);
-            return $this->sendCart($cart);
+            return $this->sendCart($cart, $storeId);
         } else {
             return $cart;
         }
     }
-
     public function sendCart(Cart $cart)
```

Comment:
We should probably only grab `storeId` once. Looks like you're passing it above (line 54) but it's not an arg for `sendCart`, and we're re-getting it on Line 62.